frigate/util.py

```diff
@@ -429,6 +430,39 @@ def copy_yuv_to_position(
     source_frame=None,
     source_channel_dim=None,
 ):
+    def crop_clear(frame, position, fill_value):
+        frame[position[1] : position[3], position[0] : position[2]] = fill_value
+
+    def assign_resized_frame(
+        destination_slice, source_slice, resize_dim, interpolation
+    ):
+        source_img = Image.fromarray(source_slice)
+        resized_img = source_img.resize(
+            (resize_dim[1], resize_dim[0]), resample=interpolation
+        )
+        destination_slice[:] = np.array(resized_img)
+
+    def resize_copy(
+        frame,
+        source_frame,
+        source_channel_dim,
+        position,
+        resize_dim,
+        offset,
+        interpolation=Image.Resampling.NEAREST,
```

Comment:
In my testing, this interpolation for a resize is faster, but the quality is really bad. I would be curious to see if there is still a performance improvement if using `Image.Resampling.BILINEAR` instead. I believe that is the equivalent to opencv's INTER_LINEAR which was used previously.

Reply:
So this is specifically for birdseye, no? Running this PR, when I only have a single camera active (which is rare), I can notice a quality degradation. Though that's possibly because my birdseye resolution (1920x1080) is bigger than my detect resolution (1280x720) that drives it. When I have multiple cameras, I can't notice poor quality at all. Perhaps if we're resizing _larger_, we use bilinear, but if we're resizing _smaller_ the faster/simpler algorithm can be used?

Reply:
> In my testing, this interpolation for a resize is faster, but the quality is really bad

describe your testing configuration